dev-util/codeblocks/codeblocks-20.03_p13518-r1.ebuild

```diff
@@ -0,0 +1,124 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+WX_GTK_VER="3.2-gtk3"
+
+inherit autotools flag-o-matic multiprocessing wxwidgets xdg
+
+REV=13518
+REV_DATE="2024-05-05 20:30:28"
+FP_NAME=fortranproject
+FP_REV=378
+
+DESCRIPTION="The open source, cross platform, free C, C++ and Fortran IDE"
+HOMEPAGE="https://www.codeblocks.org/"
+
+# svn export --ignore-externals https://svn.code.sf.net/p/codeblocks/code/trunk@${REV} codeblocks-20.03_p${REV}
+# tar -cjf codeblocks-20.03_p${REV}.tar.bz2 codeblocks-20.03_p${REV}
+#
+# svn export https://svn.code.sf.net/p/fortranproject/code/trunk@${FP_REV} fortranproject_r${FP_REV}
+# tar -cjf fortranproject_r${FP_REV}.tar.bz2 fortranproject_r${FP_REV}
+SRC_URI="
+	https://github.com/band-a-prend/gentoo-overlay/releases/download/${PN}-20.03_p${REV}/${PN}-20.03_p${REV}.tar.bz2
+	https://github.com/band-a-prend/gentoo-overlay/releases/download/${PN}-20.03_p${REV}/${FP_NAME}_r${FP_REV}.tar.bz2
+"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="fortran contrib debug"
+
+BDEPEND="virtual/pkgconfig"
+
+RDEPEND="
+	app-arch/zip
+	dev-libs/glib:2
+	>=dev-libs/tinyxml-2.6.2-r3
+	>=dev-util/astyle-3.1-r2:0/3.1
```

Comment:
Why is `dev-util/astyle` restricted to `3.1` subslot? You introduced it in commit 4103b7a5e837 - `dev-util/codeblocks: update codeblocks-9999.ebuild` but I couldn't find specifics why it is needed. All `codeblocks` versions in the tree seem to build just fine with most recent `dev-util/astyle-3.6.1:0/3.2`. It seems that `>=dev-util/astyle-3.1-r2:0=` might be more appropriate here. It would allow us to clean old `astyle` versions.

Reply:
codeblocks-20.03_p13518-r1 successfully build with `dev-util/astyle-3.6.1:0/3.2` but updated codeblocks-9999 failed to build with numerous errors related to astyle. I assume to left this changes and then explore and report to upstream.

Reply:
Add a workaround `sed -i -e '/^AX_CXX_COMPILE_STDCXX/s/11/17/' configure.ac || die` to fix codeblocks-9999 build error. It seems that an error is due to changes intorduced in revision 13577 (https://sourceforge.net/p/codeblocks/code/13577/) that adds C++17 support check.